Apps/Contoso.Forms.Demo/Contoso.Forms.Demo.Droid/Contoso.Forms.Demo.Droid.csproj

```diff
@@ -68,6 +68,8 @@
     <None Include="Resources\AboutResources.txt" />
     <None Include="Properties\AndroidManifest.xml" />
     <None Include="Assets\AboutAssets.txt" />
+    <None Include="project.json" />
```

Comment:
I think project.json being added is a merge mistake, can you check?

Reply:
good catch